src/line_range.rs

```diff
@@ -1,25 +1,35 @@
 use crate::error::*;
+use itertools::{Itertools, MinMaxResult};
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
We may not actually want to add this, since `LineRange` is `pub` and a future impl could make it not `Copy` and therefore require a breaking change.